src/python/pants/notes/2.0.x.rst

```diff
@@ -5,6 +5,108 @@ This document describes releases leading up to the ``2.0.x`` ``stable`` series.
+  `PR #10811 <https://github.com/pantsbuild/pants/pull/10811>`_
+
+Bugfixes
+~~~~~~~~
+
+* Plumb the custom certs file through to Pip (via Pex) (#10837)
+  `PR #10837 <https://github.com/pantsbuild/pants/pull/10837>`_
+
+* Fix MyPy with Python 2-only third-party requirements (#10820)
+  `PR #10820 <https://github.com/pantsbuild/pants/pull/10820>`_
+
+* Fix `setup_py.with_binaries()` to automatically inject `dependencies` (#10829)
+  `PR #10829 <https://github.com/pantsbuild/pants/pull/10829>`_
+
+* Fix run not working with foo:main entry point. (#10821)
+  `PR #10821 <https://github.com/pantsbuild/pants/pull/10821>`_
+
+* Fix `internal_only` PEXes runtime interpreter diverging from buildtime (#10818)
+  `PR #10818 <https://github.com/pantsbuild/pants/pull/10818>`_
+
+* Local process cache validates that digests exist locally before hitting (#10789)
+  `PR #10789 <https://github.com/pantsbuild/pants/pull/10789>`_
+
+* Better Ctrl-C handling in the `test --debug` case (#10835)
+  `PR #10835 <https://github.com/pantsbuild/pants/pull/10835>`_
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* OptionsBootstrapper is provided via a new SessionValues facility rather than a Param (#10827)
+  `PR #10827 <https://github.com/pantsbuild/pants/pull/10827>`_
+
+* Internally stop ignoring missing imports with MyPy (#10828)
+  `PR #10828 <https://github.com/pantsbuild/pants/pull/10828>`_
+
+* Make `get_options_to_record` a 'public' api. (#10808)
+  `PR #10808 <https://github.com/pantsbuild/pants/pull/10808>`_
+
+* Useful equality for relevant options types (#10815)
+  `PR #10815 <https://github.com/pantsbuild/pants/pull/10815>`_
+
+* Clean nodes with uncacheable dependencies once per session (#10814)
+  `PR #10814 <https://github.com/pantsbuild/pants/pull/10814>`_
+
+* Disable pantsd when piping output for internal script (#10810)
+  `PR #10810 <https://github.com/pantsbuild/pants/pull/10810>`_
+
+* Simplify logging initialization (#10812)
+  `PR #10812 <https://github.com/pantsbuild/pants/pull/10812>`_
+
+* Remove support for "weak" Gets (#10805)
+  `PR #10805 <https://github.com/pantsbuild/pants/pull/10805>`_
+
+* Un-revert Add cache_failures option to Process type (#10796)
+  `PR #10796 <https://github.com/pantsbuild/pants/pull/10796>`_
+
+* Remove reset_interactive_output_stream (#10791)
+  `PR #10791 <https://github.com/pantsbuild/pants/pull/10791>`_
+
+* Allow creating empty directories with the engine (#10760)
+  `PR #10760 <https://github.com/pantsbuild/pants/pull/10760>`_
+
+* Use "output" and "input" instead of "product" and "subject" (#10790)
+  `PR #10790 <https://github.com/pantsbuild/pants/pull/10790>`_
+
+Testing
+~~~~~~~
+
+* Re-export `QueryRule` from `pants.testutil.rule_runner` (#10806)
+  `PR #10806 <https://github.com/pantsbuild/pants/pull/10806>`_
+
+* Add graph test for UncacheableDeps nodes only running once (#10795)
+  `PR #10795 <https://github.com/pantsbuild/pants/pull/10795>`_
```

Comment:
Nit: the former change is a "testing API change", the latter is internal-only. We might be treating the definition of API Changes a bit too narrowly if the first thing doesn't fit there... maybe "API Changes" should be split into "extension API changes" and "Options changes"/"End User changes"?

Reply:
Yeah, I think we should make that distinguishment, for sure. And get rid of Testing. I've never totally understood the rationale for it being its own thing.